### PR TITLE
chore: Set Prettier to ignore integration output HTMLs

### PR DESCRIPTION
**Pull request by petermekhaeil**
Prettier is failing because it is checking the output of the integration tests. 

This commit will set Prettier to ignore those HTMLs.

## Documentation / Examples

- [x] Make sure the linting passes

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -16,3 +16,5 @@ packages/next-codemod/**/*.js
 packages/next-codemod/**/*.d.ts
 packages/next-env/**/*.d.ts
 test-timings.json
+test/integration/custom-routes/out/**/*.html
```
Comment (timneutkens) [inline .prettierignore:19]: I think a negative lookahead would be better here, `test` only holds `.js`,`.jsx`,`.ts`, and `.tsx` so it'd be useful to only match for those.

**Review by ijjk — APPROVED** [review]
This matches our `**/.next/**` ignore so looks good to me, thanks!

**Review by styfle — APPROVED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| buildDuration | 13.4s | 13.6s | ⚠️ +195ms |
| buildDurationCached | 3.1s | 3.1s | -9ms |
| nodeModulesSize | 49.5 MB | 49.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.164 | 2.194 | ⚠️ +0.03 |
| / avg req/sec | 1155.36 | 1139.3 | ⚠️ -16.06 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.202 | 1.202 | ✓ |
| /error-in-render avg req/sec | 2079.45 | 2079.52 | +0.07 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| 359.HASH.js gzip | 2.96 kB | 2.96 kB | ✓ |
| 745.HASH.js gzip | 180 B | 180 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 20.9 kB | 20.9 kB | ✓ |
| webpack-HASH.js gzip | 1.53 kB | 1.53 kB | ✓ |
| Overall change | 67.8 kB | 67.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| _app-HASH.js gzip | 803 B | 803 B | ✓ |
| _error-HASH.js gzip | 3.06 kB | 3.06 kB | ✓ |
| amp-HASH.js gzip | 554 B | 554 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.52 kB | 2.52 kB | ✓ |
| head-HASH.js gzip | 2.28 kB | 2.28 kB | ✓ |
| hooks-HASH.js gzip | 903 B | 903 B | ✓ |
| image-HASH.js gzip | 5.61 kB | 5.61 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 319 B | 319 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 19.1 kB | 19.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| _buildManifest.js gzip | 490 B | 490 B | ✓ |
| Overall change | 490 B | 490 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| index.html gzip | 531 B | 531 B | ✓ |
| link.html gzip | 544 B | 544 B | ✓ |
| withRouter.html gzip | 525 B | 525 B | ✓ |
| Overall change | 1.6 kB | 1.6 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 4 Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| buildDuration | 10.9s | 10.8s | -33ms |
| buildDurationCached | 4.2s | 4.2s | ⚠️ +36ms |
| nodeModulesSize | 49.5 MB | 49.5 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.241 | 2.24 | 0 |
| / avg req/sec | 1115.57 | 1116.21 | +0.64 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.199 | 1.206 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 2084.24 | 2072.77 | ⚠️ -11.47 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| 17.HASH.js gzip | 2.98 kB | 2.98 kB | ✓ |
| 18.HASH.js gzip | 185 B | 185 B | ✓ |
| 677f882d2ed8..HASH.js gzip | 13.7 kB | 13.7 kB | ✓ |
| framework.HASH.js gzip | 41.9 kB | 41.9 kB | ✓ |
| main-HASH.js gzip | 8.4 kB | 8.4 kB | ✓ |
| webpack-HASH.js gzip | 1.22 kB | 1.22 kB | ✓ |
| Overall change | 68.4 kB | 68.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| _app-HASH.js gzip | 791 B | 791 B | ✓ |
| _error-HASH.js gzip | 3.76 kB | 3.76 kB | ✓ |
| amp-HASH.js gzip | 552 B | 552 B | ✓ |
| css-HASH.js gzip | 333 B | 333 B | ✓ |
| dynamic-HASH.js gzip | 2.71 kB | 2.71 kB | ✓ |
| head-HASH.js gzip | 2.97 kB | 2.97 kB | ✓ |
| hooks-HASH.js gzip | 911 B | 911 B | ✓ |
| index-HASH.js gzip | 231 B | 231 B | ✓ |
| link-HASH.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..HASH.js gzip | 298 B | 298 B | ✓ |
| script-HASH.js gzip | 3.02 kB | 3.02 kB | ✓ |
| withRouter-HASH.js gzip | 294 B | 294 B | ✓ |
| e025d2764813..52f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 17.6 kB | 17.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| _buildManifest.js gzip | 500 B | 500 B | ✓ |
| Overall change | 500 B | 500 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | petermekhaeil/next.js prettier-ignore-test-htmls | Change |
| - | - | - | - |
| index.html gzip | 577 B | 577 B | ✓ |
| link.html gzip | 587 B | 587 B | ✓ |
| withRouter.html gzip | 569 B | 569 B | ✓ |
| Overall change | 1.73 kB | 1.73 kB | ✓ |

</details>

</details>
Commit: 8164c88e9305e067d9afa153e183bf88790e06d2

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 8164c88e9305e067d9afa153e183bf88790e06d2

**test/integration/build-indicator/test/index.test.js** 
- Build Activity Indicator > Enabled > Shows build indicator when page is built from modifying

<details>
<summary>Expand output</summary>

  ● Build Activity Indicator › Enabled › Shows build indicator when page is built from modifying

    expect(received).toBe(expected) // Object.is equality

    Expected: true
    Received: null

      62 |       const wasVisible = await browser.eval('window.showedBuilder')
      63 |
    > 64 |       expect(wasVisible).toBe(true)
         |                          ^
      65 |       await fs.writeFile(pagePath, origContent, 'utf8')
      66 |       await browser.close()
      67 |     })

      at Object.<anonymous> (integration/build-indicator/test/index.test.js:64:26)

</details>

**Pull request merged**